airbyte-webapp/src/packages/cloud/components/experiments/SpeedyConnection/SpeedyConnectionBanner/SpeedyConnectionBanner.module.scss

```diff
@@ -12,8 +12,8 @@
   padding: 8px;
   display: flex;
   align-items: center;
-  background-color: colors.$blue;
-  color: colors.$white;
+  background-color: colors.$blue-100;
+  color: colors.$dark-blue;
```

Comment:
The banner design was made by Nico for all our banners. I'd prefer if we don't diverge for this banner from it. In general :broom: if we keep this, we should just use the `AlertBanner` component.